azure/template.json

```diff
@@ -244,6 +244,9 @@
                     "appServicePlanResourceGroup": {
                         "value": "[resourceGroup().name]"
                     },
+                    "netFrameworkVersion": {
```

Comment:
@ChrisJWoodcock I've added this since your approval. @CloudPlatformer [asked me to add this to another PR](https://github.com/SkillsFundingAgency/das-apprentice-feedback-jobs/pull/22#issuecomment-1377704434), so I thought this one might need the same change. Could you please both review and approve, thanks.